frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.styled.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import type { MouseEventHandler } from "react";
+import { Icon } from "metabase/core/components/Icon";
+import { color } from "metabase/lib/colors";
+import type { AnchorProps } from "metabase/ui";
+import { Anchor } from "metabase/ui";
+
+export const VerticalDivider = styled.div`
+  align-self: stretch;
+  width: 1px;
+  background-color: ${color("border")};
+  padding: 5px 0px;
+`;
+
+export const ActionContainer = styled.div`
+  padding: 4px;
+  &:hover {
+    color: ${color("text-dark")};
+  }
+`;
+
+type TapButtonProps = AnchorProps & { onClick: MouseEventHandler };
+export const TabButton = styled(Anchor)<TapButtonProps>`
```

Comment:
With this one, you can set the props to
```
type TapButtonProps = AnchorProps & HTMLAttributes<HTMLAnchorElement>
```

Since the `Anchor` mantine component is based off off of `<a>`, we still get access to the React/HTML attributes here. This generally also applies to other Mantine components (i.e. `Box`<> `div`, `Text` <> `span/div`, `Button` <> `button`) - the docs go into a little more detail